src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -58,6 +58,8 @@ class PhpDumper extends Dumper
     private $targetDirRegex;
     private $targetDirMaxMatches;
     private $docStar;
+    private $serviceIdToMethodNameMap = array();
+    private $usedMethodNames = array();
```

Comment:
Yeah I meant that this parameters are different for each call to ``dump()``.
Anyway, I solved it by regenerating them each time.